main.go

```diff
@@ -60,12 +61,14 @@ func init() {
 
 func main() {
 	var enableLeaderElection bool
+	var installName string
 	var metricsAddr string
 	var probeAddr string
 	var vintageMode bool
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&installName, "install-name", "unknown", "Name of the installation")
```

Comment:
```suggestion
	flag.StringVar(&installationName, "install-name", "unknown", "Name of the installation")
```

Reply:
Done :heavy_check_mark: 
BTW: that's nice of you to do suggestions, but when they require updates in other parts of the code that looks like a trap :rofl:

Reply:
True but I'd rather not do suggestions all around :D